io.openems.edge.evcs.cluster/bnd.bnd

```diff
@@ -0,0 +1,26 @@
+Bundle-Name: OpenEMS Edge EVCS Cluster
+Bundle-Vendor: OpenEMS Association e.V.
+Bundle-License: https://opensource.org/licenses/EPL-2.0
+Bundle-Version: 1.0.0.${tstamp}
+Bundle-Description: \
+	The provider bundle for io.openems.edge.evcs.cluster \
+	\
+Export-Package:  \
+	io.openems.edge.controller.evcs.collection.api;-provide=true
+
+Private-Package: io.openems.edge.evcs.cluster
+
+-includeresource: {readme.md}
```

Comment:
Please provide 'readme.adoc' file, describing the EVCS cluster; at least once mention what "EVCS" stands for.